luma/core/interface/parallel.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+"""
+Encapsulates sending commands and data over a parallel-bus interface.
+"""
+
+from time import sleep
+from luma.core import lib
+
+
+__all__ = ["bitbang_6800"]
+
+#: Default amount of time to wait for a pulse to complete if the device the interface is connected to requires a pin to be 'pulsed' from low to high to low for it to accept data or a command
+PULSE_TIME = 1e-6 * 50
+
+
+@lib.rpi_gpio
+class bitbang_6800(object):
+    """
+    Implements a 6800 style parallel-bus interface that provides :py:func:`data`
+    and :py:func:`command` methods. The default pin assignments provided are
+    from `Adafruit <https://learn.adafruit.com/drive-a-16x2-lcd-directly-with-a-raspberry-pi/wiring>`.
+
+    :param gpio: GPIO interface (must be compatible with
+        `RPi.GPIO <https://pypi.python.org/pypi/RPi.GPIO>`_)
+    :param pulse_time: length of time in seconds that the enable line should be
+        held high during a data or command transfer
+    :type pulse_time: float
+    :param RS: The GPIO pin register select (RS) line (low for command, high
+        for data)
+    :type RS: int
+    :param E: The GPIO pin to connect the enable (E) line to.
+    :type E: int
+    :param PINS: The GPIO pins that form the data bus (a list of 4 or 8 pins
+        depending upon implementation ordered from LSD to MSD)
+    :type PINS: list[int]
+
+    .. versionadded:: 1.15.1
```

Comment:
i would update the version nr and use latest

Reply:
or leave it as is and add a version marker in the module docstring with latest

Reply:
Changed to 1.16.2